install_config/http_proxies.adoc

```diff
@@ -35,7 +38,7 @@ endif::[]
 ----
 HTTP_PROXY=http://USERNAME:PASSWORD@10.0.1.1:8080/
 HTTPS_PROXY=https://USERNAME:PASSWORD@10.0.0.1:8080/
-NO_PROXY=master.hostname.example.com
+NO_PROXY=master.hostname.example.com,10.1.0.0/16,172.30.0.0/16
```

Comment:
@deads2k Thanks. I meant instead of up in the Overview (it's in both places currently). It seemed strange to specifically mention NO_PROXY in the Overview when we have a place we can go into detail about it later when it's used/needed.